knack/iframeMapMessenger.js

```diff
@@ -64,61 +84,66 @@
       var recordId = urlArray[urlArray.length - 2];
       var workOrderId = urlArray[urlArray.length - 4];
 
-      var markerMessage = {
+      var headers = getHeaders(Knack.getUserToken(), Knack.application_id);
+      var signsMarkerMessage = {
         message: "KNACK_LOCATION_DETAILS",
-        view: "view_2733",
-        scene: "scene_1039",
-        token: Knack.getUserToken(),
-        app_id: Knack.application_id,
-        id: recordId,
-        workOrderScene: "scene_1028",
-        workOrderId: workOrderId,
-        workOrderView: "view_2573"
```

Comment:
i really like that you moved the data fetching elsewhere and simplified the message payload 👍